maintainer/benchmarks/ferrofluid.py

```diff
@@ -101,111 +91,65 @@
 system.cell_system.skin = 0.5
 system.thermostat.turn_off()
 
-
-#############################################################
-#  Setup System                                             #
-#############################################################
-
 # Interaction setup
 #############################################################
 system.non_bonded_inter[0, 0].lennard_jones.set_params(
     epsilon=lj_eps, sigma=lj_sig, cutoff=lj_cut, shift="auto")
 
-print("LJ-parameters:")
-print(system.non_bonded_inter[0, 0].lennard_jones.get_params())
-
 # Particle setup
 #############################################################
+system.part.add(
+    pos=np.random.random((n_part, 3)) * system.box_l,
```

Comment:
remove the comment line 84-86 as in the other benchmarks
```
# PRNG seeds
#############################################################
# np.random.seed(1)
```

Reply:
good catch!